hw1/02-palindrome.js

```diff
@@ -1,3 +1,27 @@
-const elem = document.querySelector('input');
+const elem = document.querySelector("input");
```

Comment:
Airbnb's style guide requires using single quotes rather than double quotes. This change will need to be made throughout your file, but I'm just going to comment it here.

Reply:
Thank you, I will fix that right away.

Reply:
I have fixed this issue in a recent commit.